deployment/envgen2/EnvGen.cpp

```diff
@@ -933,7 +942,7 @@ void CEnvGen::addUpdateTaskFromFile(const char * inFile)
    }
```

Comment:
Should 
  if (!(*propName)) continue;
Be
  if ( (propName != nullptr) && *propName) continue;

Reply:
It checks propName string is not empty string. I really reply on PropertyTreeIterator to throw exception if no property name item available